Sources/SwiftProtobuf/BinaryEncoder.swift

```diff
@@ -123,8 +128,8 @@ internal struct BinaryEncoder {
         let count = value.utf8.count
```

Comment:
Does it make sense to attempt to use withContiguousStorageIfAvailable to fast path the whole string in here and then fall back to this loop if we can't?

Reply:
Yes, this is a good enhancement. For the sake of not boiling the ocean I'd like not to make it here: may I suggest filing an issue for this? It could even be marked contributor friendly and a new contributor could take a swing at it, as it should be easy enough to do.

Reply:
Opened.